src/server/index.js

```diff
@@ -17,6 +17,8 @@ import handlePurge from './handle-purge'
 import handleRedirects from './handle-redirects'
 import CacheManager from '../utilities/cache-manager'
 
+const cacheManager = new CacheManager
```

Comment:
Yes. I also don't really like the implementation of new.

I just want to import `cacheManager from 'cacheManager'`... which might work now actually.